packages/components/src/SubHeaderBar/InputTitleSubHeader/InputTitleSubHeader.scss

```diff
@@ -35,6 +34,7 @@ $tc-input-subheader-size-small: 1rem !default;
 		padding-left: $padding-large;
 		overflow: hidden;
 		flex: 1;
+		margin-right: 110px;
```

Comment:
this is to remove the fixed with. So we are sure when swithing from display to inline form to not change the width.